datanode/sqlstore/migrations/0057_transfers_fees_discount.sql

```diff
@@ -0,0 +1,22 @@
+-- +goose Up
+
+ALTER TABLE transfer_fees ADD COLUMN discount_applied HUGEINT;
+
+-- transfer_fees_discount table contains the per party, per party available transaction fee discount.
+CREATE TABLE IF NOT EXISTS transfer_fees_discount (
```

Comment:
Is this something we need a hypertable for? I know @guoguojin has just fixed some other tables that we forgot to hyper.

Reply:
Looking at it, I'm going to be the man from Del Monte and say, "Yes!"